src/aliases/models/alias.go

```diff
@@ -0,0 +1,20 @@
+package aliasmodels
+
+// Alias allows the user to associates a RegistryName + a Key to 1 or more public keys stored
+// in Value. The Value has 2 formats:
+// - a JSON string if AliasKind is an AliasKindString.
+// - a JSON array of strings if AliasKind is an AliasKindArray.
+type Alias struct {
+	tableName struct{} `pg:"aliases"` // nolint:unused,structcheck // reason
+
+	Key          AliasKey     `pg:",pk"`
```

Comment:
This model does not correlate to your PG table definition where pk is the a serial id, what is the reason for this?

Reply:
forgot to fix the migration script.

It is fixed now.